client/Android/android_freerdp.c

```diff
@@ -292,6 +292,7 @@ static BOOL android_register_pointer(rdpGraphics* graphics)
 	if (!graphics)
 		return FALSE;
 
+	memset(&pointer, 0, sizeof(rdpPointer));
```

Comment:
nice find, thank you for the pr.
We´d prefer the `rdpPointer pointer = { 0 };` syntax though, I´ll do that for you.